README.md

```diff
@@ -1,75 +1,36 @@
-# Collector
+# Stackrox Collector
 
-## Collector development
+![GitHub branch checks state](https://img.shields.io/github/checks-status/stackrox/collector/master?style=for-the-badge)
+![GitHub](https://img.shields.io/github/license/stackrox/collector?style=for-the-badge)
+![GitHub tag (latest SemVer)](https://img.shields.io/github/v/tag/stackrox/collector?sort=semver&style=for-the-badge)
 
-### Cloning the repo
+Welcome to the Stackrox Collector project documentation. Here you can learn
+more about idea behind the project, how to start guidelines, design overview
+and detailed references.
 
-Since the collector repo uses a submodule for the sysdig source, use
-the following command to clone.
+Collector is a component of Stackrox responsible for gathering runtime data. In
+a few words it is an agent that runs on every node under strict performance
+limitations and gathers the data via kernel modules or eBPF probes (the default
+collection mode nowadays). To implement eBPF probes and collect data, the
+project leverages the Falco libraries via a custom
+[fork](https://github.com/stackrox/falcosecurity-libs).
 
-```
-git clone --recurse-submodules git@github.com:stackrox/collector.git
-```
+There are opportunities to catch in many areas, so let's start.
```

Comment:
What is the meaning of catch here?

Reply:
Meaning if somebody wants to contribute, there are opportunities for that. It sounded a bit awkward, so I've changed it a bit.